Makefile.jenkins

```diff
@@ -88,6 +88,11 @@ endif
 
 ui-build:
 	rm -fr build/*
+	# If NPM_CONFIG_REGISTRY is set, change yarn.lock to use that mirror.
+	if [ -n "$$NPM_CONFIG_REGISTRY" ]; then \
+	  sed -i -e "s#https://registry.yarnpkg.com/#$$NPM_CONFIG_REGISTRY/#g" yarn.lock; \
+	  sed -i -e "s#https://registry.npmjs.org/#$$NPM_CONFIG_REGISTRY/#g" yarn.lock; \
```

Comment:
Do we have `npmjs` links in `yarn.lock` ?

Reply:
Nope. That is just in case :)